_pages/nt7ab.yaml

```diff
@@ -0,0 +1,6 @@
+---
+githubHandle: nt7ab
+pageUrl: https://nt7ab.github.io/
+timestamp: 2017-01-18
+---
+
```

Comment:
@NehaTelhan since you've introduced the other file, you can remove this file altogether